CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## v0.52.0 [2024-05-13]
+
+_Enhancements_
+
+- The Plugin and the Steampipe Anywhere binaries are now built with the `netgo` package. ([#580](https://github.com/turbot/steampipe-plugin-gcp/pull/580))
```

Comment:
@khushboo9024 could you please add the note for the column that was converted to a connection qual in this case.